src/main/webapp/Localization/localize_de.properties

```diff
@@ -0,0 +1,3 @@
+How\ to\ localize\ texts\ in\ Jelly=So lokalisieren Sie Texte in Jelly
+longtext=Ein l�ngerer Text, der nicht inline in der Jelly-Datei erscheinen soll.
```

Comment:
the character appears to be invalid (and renders wrong for me)

<img width="801" alt="image" src="https://github.com/user-attachments/assets/7b61583a-acd4-4a20-8a0e-915b9205448d" />

Reply:
fixed